projects/fsastorefront-e2e-cypress/cypress/helpers/search.ts

```diff
@@ -1,57 +1,61 @@
 export function searchNoResults() {
-  cy.get('cx-searchbox')
+  cy.get('cx-fs-searchbox')
     .should('be.visible')
     .within(() => {
-      cy.get('input').type('notexist{enter}').wait(1500);
+      cy.get('input').type('notexist{enter}');
     });
   cy.get('cx-breadcrumb').should('contain', '0 results for "notexist"');
 }
 
 export function searchInsuranceProducts() {
-  cy.get('cx-searchbox')
+  cy.get('cx-fs-searchbox')
     .should('be.visible')
     .within(() => {
-      cy.get('input').clear().type('insurance{enter}').wait(1500);
+      cy.get('input').clear().type('insurance{enter}');
     });
   cy.get('cx-breadcrumb').should('contain', '25 results for "insurance"');
 }
 
 export function searchBankingProducts() {
-  cy.get('cx-searchbox')
+  cy.get('cx-fs-searchbox')
     .should('be.visible')
     .within(() => {
-      cy.get('input').clear().type('account{enter}').wait(1500);
+      cy.get('input').clear().type('account{enter}');
     });
   cy.get('cx-breadcrumb').should('contain', '3 results for "account"');
 }
 
-export function searchSavingsProducts() {
-  cy.get('cx-searchbox')
+export function searchTravelProducts() {
```

Comment:
Why travel instead savings?

Reply:
Why Savings instead of Travel? I like Travel more...
Savings and Life category pages have changed and those changes would require more refactoring then just switching to Travel category.